### PR TITLE
qa/workunits/rados: wait 60 seconds for ceph-crash to restart

### PR DESCRIPTION
**Pull request by ljflores**
Let's give ceph-crash a full 60 seconds to restart the osds. If we end up checking for crashes before the osds have fully restarted, we will fail the test.

Fixes: https://tracker.ceph.com/issues/58098
Signed-off-by: Laura Flores <lflores@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by ljflores** [comment]
jenkins test make check

**Comment by ljflores** [comment]
Running some tests here:
http://pulpito.front.sepia.ceph.com/lflores-2022-11-30_16:20:41-rados-main-distro-default-smithi/

File: qa/workunits/rados/test_crash.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ done
 
 # let daemon find crashdumps on startup
 sudo systemctl restart ceph-crash
-sleep 30
+sleep 60
```
Comment (neha-ojha) [inline qa/workunits/rados/test_crash.sh:29]: How reproducible is this https://tracker.ceph.com/issues/58098? We should try to understand why the existing sleep value isn't enough any longer so as to not mask other issues with slow OSD restarts.

Comment (ljflores) [reply qa/workunits/rados/test_crash.sh:29]: I'll run some tests to try and reproduce the issue, and mark this PR as a draft for now.

**Pull request closed without merging**